website/source/docs/config/index.html.md

```diff
@@ -90,6 +93,18 @@ For Consul, the following options are supported:
 
   * `token` (optional) - An access token to use to write data to Consul.
 
+#### Backend Reference: Zookeeper
+
+For Zookeeper, the following options are supported:
+
+  * `path` (optional) - The path within Zookeeper where data will be stored.
+      Defaults to "vault/".
+
+  * `address` (optional) - The address(es) of the Zookeeper instance(s) to talk to.
+      Can be comma separated list of many Zookeeper instances.
+      Defaults to the local agent address, if available.
```

Comment:
Does this default come from the library? I don't see the address being defaulted here is all